packages/core/admin/admin/src/content-manager/components/NonRepeatableComponent/index.js

```diff
@@ -21,6 +22,8 @@ const NonRepeatableComponent = ({ componentUid, isFromDynamicZone, isNested, nam
 
   const { lazyComponentStore } = useLazyComponents();
 
+  const isFromDynamicZone = source.type === ItemTypes.DYNAMIC_ZONE;
```

Comment:
I'm not sure this makes a lot of sense because you only need to know if its in a dynamic zone not if it's in a repeatable component? So maybe `source.type` is unneccessary?